extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -771,6 +771,13 @@ tags: {  # <amp-story-page-attachment>
     value: "dark"
     value: "light"
   }
+  attrs: {
+    name: "href"
+    value_url: {
+      protocol: "http"
+      protocol: "https"
+    }
+  }
```

Comment:
Having any descendant today breaks the layout. I think we should have two separate validation specs for this `<amp-story-page-attachment>` tag:
1. The current one that allows descendants
2. A new one with a mandatory `href` attribute, and 0 allowed descendants

The alternative would be to remove all descendants at build time, but the validation spec is more strict.

Reply:
Done.

Reply:
Can you push the changes?

Reply:
Sorry. Done